pkgs/tools/networking/wireguard-go/deps.nix

```diff
@@ -1,30 +1,57 @@
 # file generated from go.mod using vgo2nix (https://github.com/adisbladis/vgo2nix)
 [
+  {
+    goPackagePath = "github.com/Microsoft/go-winio";
+    fetch = {
+      type = "git";
+      url = "https://github.com/Microsoft/go-winio";
```

Comment:
Looks like vgo2nix doesn't respect go.mod's replace operator. Shouldn't be an issue in this case, but it is notable.

Reply:
Could you open an issue at https://github.com/adisbladis/vgo2nix for that?

I lack the go skills to investigate this further.

Reply:
Opened issue upstream: https://github.com/adisbladis/vgo2nix/issues/22